test-jest/polar/Pie.spec.tsx

```diff
@@ -25,19 +21,27 @@ describe('<Pie />', () => {
     { cx: 250, cy: 250, innerRadius: 50, outerRadius: 100, startAngle: 288, endAngle: 360 },
   ];
 
-  it('Render 5 sectors in a simple Pie', () => {
-    const wrapper = render(
+  test('Render 5 sectors in a simple Pie', () => {
+    const { container } = render(
       <Surface width={500} height={500}>
-        <Pie isAnimationActive={false} cx={250} cy={250} innerRadius={0} outerRadius={200} sectors={sectors} />
+        <Pie
+          isAnimationActive={false}
+          cx={250}
+          cy={250}
+          innerRadius={0}
+          outerRadius={200}
+          sectors={sectors}
+          dataKey="cy"
+        />
       </Surface>,
     );
 
-    expect(wrapper.find('.recharts-pie-sector').length).to.equal(sectors.length);
+    expect(container.querySelectorAll('.recharts-pie-sector').length).toBe(sectors.length);
   });
 
-  it('Render customized active sector when activeShape is set to be an element', () => {
-    const ActiveShape = props => <Sector {...props} fill="#ff7300" className="customized-active-shape" />;
-    const wrapper = render(
+  test('Render customized active sector when activeShape is set to be an element', () => {
+    const ActiveShape = (props: any) => <Sector {...props} fill="#ff7300" className="customized-active-shape" />;
```

Comment:
please avoid any where possible/if possible (might not be easy to here, but just a general comment)

Reply:
I added some additional types for the `label` and `labelLine` so we don't have to use `any`.

But the `activeShape` and `inactiveShape` both have types `((props: any) => ReactElement<SVGElement>)` so I have left these with `any` to match this type.

Reply:
Thanks!